.eslintignore

```diff
@@ -0,0 +1,2 @@
+*.test.js
+*.stories.js
```

Comment:
Just wanted to check in on what your thoughts were around excluding test and storie files from eslint?